### PR TITLE
feat: event attendence

### PR DESCRIPTION
**Pull request by mavehur**

### CODE REVIEW

File: app/event-attendance/EventAttendance.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```
Comment (rustamch) [inline app/event-attendance/EventAttendance.tsx:1]: I believe that we need to include 'useClient' here.

File: app/event-attendance/page.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import React, { useState } from 'react';
```
Comment (rustamch) [inline app/event-attendance/page.tsx:1]: I believe that we need to either move the useState to Component or add 'useClient' here.

Comment (steph-feng) [reply app/event-attendance/page.tsx:1]: i agree, other than that, looks good to me

Comment (janayee-c) [reply app/event-attendance/page.tsx:1]: also just getting this error, but other than that looks clean !

**Comment by mavehur** [comment]
Added `user client ` as suggested from this [PR](https://github.com/ubccsss/Blood-For-Life-Management-Frontend/pull/2)

File: app/event-attendance/page.tsx (hunk 2 above)
Comment (mavehur) [reply app/event-attendance/page.tsx:1]: Oh yes. Just added `use client` as suggested!

Comment (mavehur) [reply app/event-attendance/page.tsx:1]: Closed and opened a new PR!

**Pull request closed without merging**